controllers/account/ec2.go

```diff
@@ -127,6 +127,14 @@ func (r *AccountReconciler) InitializeRegion(
 		return err
 	}
 
+	// Skip region initialization for GovCloud as it is always BYOVPC and never non-CCS
+	// Customers in FedRAMP often do not have quota for extra VPCs
+	if config.IsFedramp() {
```

Comment:
We could maybe do this even before loading credentials, but I like that we at least get to validate them in this func, so seems fine 👍